x-pack/plugin/esql/src/test/java/org/elasticsearch/xpack/esql/analysis/AnalyzerTestUtils.java

```diff
@@ -44,6 +47,10 @@ public static Analyzer analyzer(IndexResolution indexResolution, Verifier verifi
         );
     }
 
+    public static Analyzer analyzer(EsqlConfiguration config, IndexResolution indexResolution, Verifier verifier) {
```

Comment:
nit: argument order breaks the telescope pattern; would be slightly more expected if this was
```suggestion
    public static Analyzer analyzer(IndexResolution indexResolution, Verifier verifier, EsqlConfiguration config) {
```